using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -841,7 +841,7 @@ ACPTarget.retrieveLocationContent(requestArray, with: targetParameters)
 
 ### retrieveLocationContent
 
-This API will send a batch request to configured Target server only for mbox locations in the `TargetRequest` list that are not already prefetched. The content for the mbox locations which are already prefetched with a prior prefetch request, will be returned from within the SDK itself. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
+For mbox locations in the `TargetRequest` list that are not already prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK and no additional network request is made. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```

Comment:
in the 2nd sentence, please add a comma between "SDK" and "and"

Reply:
I added the missing commas in this commit: https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/207/commits/4534de311a084c6f00d938ea0dcda6e650b39b62

Reply:
I added the missing commas in this commit: https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/207/commits/4534de311a084c6f00d938ea0dcda6e650b39b62